README.md

```diff
@@ -6,6 +6,7 @@ Frontend rendering framework for theguardian.com. It uses [React](https://reactj
 <!-- DON'T EDIT THIS SECTION, INSTEAD RE-RUN doctoc TO UPDATE -->
 <!-- Automatically created with yarn run createtoc and on push hook -->
 
+- [Moving to main as default branch](#moving-to-main-as-default-branch)
```

Comment:
Worth rebasing?

Reply:
I've tried that but seems like that change is still there. I think it is auto-generated via `yarn run` to provide a quick link of the new section `Moving to main as default branch`

Reply:
looks like I cannot even revert it. I committed a revert but an autogenerated commit called `Add TOC update` followed straight away and put it back 😅